library/network/src/modules/SuSEFirewall.rb

```diff
@@ -154,9 +154,64 @@ def self.create
   # ----------------------------------------------------------------------------
   # SuSEFirewalld Class. Trying to provide relevent pieces of SF2 functionality via
   # firewalld.
-  class SuSEFirewalld < Module
+  class SuSEFirewalld < Firewall
+    attr_reader :special_all_interface_zone
+
+    # Valid attributes for firewalld zones
```

Comment:
`modified` is also an Array of symbols which is used to indicate which settings have been modified. For example
`[:interfaces, :ports]` so we only make the necessary calls to firewall-cmd